README.md

```diff
@@ -14,13 +14,19 @@ This Repository aims to provide various SVG icons to use in your Readme, or any
 
 ## Table of Contents
 
-- [How to use](#how-to-use)
-  - [Specific versions](#specific-versions)
-- [Licenses](#licenses)
-- [Why use this and not the actual source?](#why-use-this-and-not-the-actual-source)
-- [Icons](#icons)
-  - [Octicons](#octicons)
-- [Submit Icons](#submit-icons)
+- [Readme-Icons](#readme-icons)
```

Comment:
Andre will not accept `#readme-icons`, Don't remove until @Andre601 will say something.

Reply:
Already gave my feedback.